components/sup/src/manager/mod.rs

```diff
@@ -42,7 +42,7 @@ use hcore::fs::FS_ROOT_PATH;
 use hcore::service::ServiceGroup;
 use hcore::os::process::{self, Signal};
 use hcore::package::{Identifiable, PackageIdent, PackageInstall};
-use launcher_client::LauncherCli;
+use launcher_client::{LAUNCHER_LOCK_CLEAN_ENV, LAUNCHER_PID_ENV, LauncherCli};
```

Comment:
Yeah I like this pattern a lot. It lets the Supervisor completely ignore the protocol crate :)